spring-integration-azure/spring-integration-storage-queue/src/test/java/com/microsoft/azure/spring/integration/storage/queue/StorageQueueTemplateReceiveTest.java

```diff
@@ -60,7 +60,7 @@ public void testReceiveFailure() throws StorageException {
                 .thenThrow(StorageException.class);
 
         CompletableFuture<Message<?>> future =
-                this.operation.receiveAsync(this.destination, this.visibilityTimeoutInSeconds);
+                this.operation.receiveAsync(this.destination);
```

Comment:
looks can be put it in one line.

Reply:
yes, will handle format in another change.